doc/Security.xml

```diff
@@ -986,7 +1075,7 @@ Added certificate-based client authentication</revremark>
       <section>
         <title>Elements of the Keystore</title>
         <para>The keystore security feature handles the storage and management of passphrases, keys, and certificates on an ONVIF device.</para>
-        <para>The keystore specified in this document supports passphrases, keys, key pairs, which are a particular type of key, RSA key pairs, which are a particular type of key pairs, certificates, certification paths, certificate revocation lists, and certification path validation policies.</para>
+        <para>The keystore specified in this document supports passphrases, keys, key pairs, which are a particular type of key, RSA and ECC key pairs, which are a particular type of key pairs, certificates, certification paths, certificate revocation lists, and certification path validation policies.</para>
```

Comment:
Suggested Update 

"which are a particular type of key, RSA and ECC key pairs, also a particular key pairs type,"

I kept reading the document and my mind could not get past this sentence without a red flag.